spec/components/previews/layouts/component_preview.html.erb

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <% content_for :error_tracking do %>
+      <%= render 'layouts/error_tracking' %>
+    <% end %>
+    <title>
+      <%= content_for(:title) || 'Your Career in Web Development Starts Here | The Odin Project' %>
+    </title>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="The Odin Project empowers aspiring web developers to learn together for free">
+    <meta property="og:image" content="<%= image_url('og-logo.png') %>">
+    <%= csrf_meta_tags %>
+
+    <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png">
+    <link rel="icon" type="image/png" href="/favicon-32x32.png" sizes="32x32">
+    <link rel="icon" type="image/png" href="/favicon-16x16.png" sizes="16x16">
+    <link rel="manifest" href="/manifest.json">
+    <link rel="mask-icon" href="/safari-pinned-tab.svg" color="#5bbad5">
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,300;0,400;0,500;0,600;0,700;0,800;1,300;1,400;1,500;1,600;1,700;1,800&display=swap" rel="stylesheet">
+
+    <%= stylesheet_link_tag 'application_stylesheet', media: 'all'  %>
+    <%= stylesheet_pack_tag 'application' %>
+
+    <%= javascript_pack_tag 'application', 'data-turbo-track': 'reload', defer: true %>
+  </head>
+
+  <body>
```

Comment:
Attempted to center components in the layout, but Tailwind classes weren't getting picked up from here. Didn't use the default layout because there were a number of dependencies (such as current theme, current user)

Reply:
Adding the previews path to [this list](https://github.com/TheOdinProject/theodinproject/blob/main/tailwind.config.js#L4) should hopefully solve that 🤞